packages/app/package.json

```diff
@@ -3,14 +3,15 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
+    "prebuild": "cd ../contracts && npm run build",
```

Comment:
we need the contracts to be built first so that the app could use the ABIs. But vercel seems like it's still failing prob bc of using a relative path. Could I get access to the Vercel logs?